charts/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -141,6 +141,36 @@ spec:
 {{- if .Values.controller.extraContainers }}
       {{ toYaml .Values.controller.extraContainers | nindent 6 }}
 {{- end }}
+
```

Comment:
same is required for daemonset template as well

Reply:
I'll re-run tests using Daemonset as well. Nice catch.